modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Installed the OpenShift CLI (`oc`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+az login
+----
+. Create a Microsoft storage accounts and upload the `arm64` virtual hard disk (VHD) (FIXME: To where?)
```

Comment:
The doc you sent me just said upload the `arm64` VHD but I wasn't sure on where its suppose to uploaded to? Do you mean download?

Reply:
upload the vhd to the storage account i meant. sorry the wording wasn't clear